setup.py

```diff
@@ -7,12 +7,7 @@
     author_email="hello@qiskit.org",
     url="https://github.com/Qiskit/qiskit_sphinx_theme",
     description="A Sphinx theme for Qiskit that is based on the Pytorch Sphinx theme.",
-    packages=[
-        "qiskit_sphinx_theme",
-        "qiskit_sphinx_theme.static.css",
-        "qiskit_sphinx_theme.static.images",
-        "qiskit_sphinx_theme.static.js",
-    ],
+    packages=find_namespace_packages(exclude=["docs*", "docs_guide*"]),
```

Comment:
I'd personally err on the side of "include" rather than "exclude": it's not clear to me whether "tests" is going to get pulled in here (I suspect yes, it will), and whether that was intended.  If it _was_ intended, I think it might be a mistake to include it as an explicit package: that will cause it to get installed into `site-packages` and likely have naming clashes, whereas if we _do_ want to distribute the tests, they probably only should be in the sdist (which can be done with a suitable graft line in `MANIFEST.in`, I believe).

Reply:
100% agreed. When I tried using `include` earlier today, it was still for some reason including `docs/` and `docs_guide`. But I can't reproduce that anymore, so it was probably bad leftover state or I was messing something up.

This works now. FYI tested by `rm -rf dist; python -m build; cd dist; unzip qiskit_sphinx_theme-1.11.0rc1-py2.py3-none-any.whl`, then inspecting the RECORD.

Reply:
nice, thanks for the check too 👍